package.json

```diff
@@ -16,12 +16,13 @@
     "@asyncapi/modelina": "^1.9.0",
     "@asyncapi/openapi-schema-parser": "^3.0.3",
     "@asyncapi/optimizer": "^0.2.1",
-    "@asyncapi/parser": "^2.1.0",
+    "@asyncapi/parser": "^3.0.0-next-major-spec.1",
     "@asyncapi/raml-dt-schema-parser": "^4.0.3",
     "@asyncapi/studio": "^0.17.3",
     "@oclif/core": "^1.26.2",
     "@oclif/errors": "^1.3.6",
     "@oclif/plugin-not-found": "^2.3.22",
+    "@smoya/multi-parser": "^2.0.0",
```

Comment:
This lib can eventually go into AsyncAPI org, it is just a matter of starting the "donation" process.